api/submission/submit.go

```diff
@@ -46,6 +48,10 @@ func HandleSubmit(request *SubmitRequest) (*SubmitResponse, *core.APIError) {
 
         response.Rejected = true;
         response.Message = reject.String();
+
+        if _, ok := reject.(*grader.RejectMissingLateAcknowledgment); ok {
+            response.RequireLateAcknowledgment = true;
+        }
```

Comment:
Add a newline after this if.
I don't want the return visually attached to it.

Reply:
Fixed in e79974b2e3275d2c07527dc51f08f50118b472d4

I hope this is what you meant. Let me know if I misunderstood your comment.